### PR TITLE
Abstracts for chapters 6, 8, and 10

### PR DESCRIPTION
**Pull request by bast**
Ready to go but possibly too brief?

### CODE REVIEW

File: chapter-06/recipe-05/abstract.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The goal of this recipe is similar to [Recipe 4](../recipe-04), but the
```
Comment (robertodr) [inline chapter-06/recipe-05/abstract.md]: Can you add recipe title in italics?

File: chapter-06/recipe-07/abstract.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+In [Recipe 6](../recipe-06), we recorded the state of the code repository
```
Comment (robertodr) [inline chapter-06/recipe-07/abstract.md]: Can you add the recipe title in italics?

File: chapter-08/recipe-01/abstract.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+This recipe introduces the superbuild pattern with a very simple example. We
+will show how to use the
+[ExternalProject_Add](https://cmake.org/cmake/help/latest/module/ExternalProject.html)
```
Comment (robertodr) [inline chapter-08/recipe-01/abstract.md]: Can you set the command in monospace font?

File: chapter-08/recipe-04/abstract.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+In this recipe we reuse the code from [Recipe 3 in Chapter 4](../../chapter-04/recipe-03)
+and fetch and build the [Google Test](https://github.com/google/googletest) framework
+using [FetchContent](https://cmake.org/cmake/help/latest/module/FetchContent.html),
```
Comment (robertodr) [inline chapter-08/recipe-04/abstract.md]: Can you set the commands in monospace?

File: chapter-08/recipe-05/abstract.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+This recipe shows how to use
+[ExternalProject_Add](https://cmake.org/cmake/help/latest/module/ExternalProject_Add.html)
```
Comment (robertodr) [inline chapter-08/recipe-05/abstract.md]: Can you set the command in monospace?

File: chapter-06/recipe-02/README.md [hunk 6]
```diff
@@ -1,20 +1,28 @@
 # Generating source code at configure time using Python
 
 In this recipe, we generate `print_info.c` from the template `print_info.c.in`
-by emulating the CMake function `configure_file` with a custom Python script.
+by emulating the CMake function
+[configure_file](https://cmake.org/cmake/help/latest/command/configure_file.html)
```
Comment (robertodr) [inline chapter-06/recipe-02/README.md]: Can you set the commands in monospace?

**Comment by bast** [comment]
Cheers for the corrections! I will fix that.

**Pull request merged**